### PR TITLE
Adds page for aria-meter

### PR DESCRIPTION
**Pull request by rachelandrew**
Part of #8610 

Adds a page for meter, and also updates release notes.

### CODE REVIEW

File: files/en-us/web/accessibility/aria/roles/meter_role/index.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  - ARIA
+  - Accessibility
+  - Meter
+  - Web Development
+---
+<p>The <code><a href="https://www.w3.org/TR/wai-aria-practices-1.2/#meter">meter</a></code> role is used to identify an element being used as a meter.</p>
+
+<h2 id="Description">Description</h2>
+
+<p>A meter is a graphical display of a numeric value within a defined range. For example, showing battery percentage. A meter is not appropriate for values that do not have a meaningful maximum limit. Meters should not be used to indicate progress (for example loading), this should be communicated with the <code>progress</code> role.</p>
```
Comment (ericwbailey) [inline files/en-us/web/accessibility/aria/roles/meter_role/index.html:14]:
```suggestion
<p>A meter is a graphical display of a numeric value within a defined range. For example, showing battery percentage. A meter is not appropriate for values that do not have a meaningful maximum limit. Meters should not be used to indicate progress (for example loading), this should be communicated with the <code>progress</code> element.</p>
```

What do you think about recommending the `progress` element instead? I was thinking it'd be better to defer to HTML when there already is an element that has the same AT mappings under the hood.

**Comment by ddbeck** [comment]
Hi @estelle, I thought since you were already working on this sort of thing, you might like to review this. If not, feel free to kick it back to me

**Review by estelle — APPROVED** [review]
Looks good! I'll convert it to MD once merged.

**Comment by estelle** [comment]
The broken links to roles will be fixed with the ARIA roles PR I am currently working on.

**Pull request merged**